stdlib/2and3/optparse.pyi

```diff
@@ -111,10 +111,10 @@ class Option:
     def _check_const(self) -> None: ...
     def _check_dest(self) -> None: ...
     def _check_nargs(self) -> None: ...
-    def _check_opt_strings(self, opts: Optional[_Text]) -> Any: ...
+    def _check_opt_strings(self, opts: Sequence[_Text]) -> Any: ...
```

Comment:
Actually, `Iterable` is enough. But it explicitly allows `None` values (see the comment in the implementation), so we still need `Optional`.

Reply:
That's right; will fix this.
I also noticed now that the return type could be set to `List[_Text]`.